### PR TITLE
funannotate_annotate mv: target 'out.tbl' is not a directory - delete part files in large output datasets

### PR DESCRIPTION
**Pull request by TomHarrop**
Potential workaround for #5527 

---

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
Please bumb the tool version :-)

**Comment by abretaud** [comment]
Ah nice, thanks for the debugging! Your solution looks good :+1:

File: tools/funannotate/funannotate_annotate.xml [hunk 1]
```diff
@@ -71,6 +71,12 @@ funannotate annotate
 
 &&
 
+find output/annotate_results
```
Comment (bernt-matthias) [inline tools/funannotate/funannotate_annotate.xml:74]: Would be nice to add a comment why this is needed.. Maybe report upstream.. And put a link to the upstream issue in the comment..

Comment (TomHarrop) [reply tools/funannotate/funannotate_annotate.xml:74]: Thanks, someone had already reported it but I've added a comment https://github.com/nextgenusfs/funannotate/issues/777

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**